src/improvements/tasks/MultisigTask.sol

```diff
@@ -620,28 +649,31 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager, TaskManage
     /// @notice To show the full transaction trace in Tenderly, we build custom calldata
     /// that shows both the child multisig approving the hash, as well as the parent multisig
     /// executing the task. This is only used when simulating a nested multisig.
-    function getNestedSimulationMulticall3Calldata(Action[] memory actions) internal view returns (bytes memory data) {
+    function getNestedSimulationMulticall3Calldata(address[] memory allSafes, bytes[] memory allCalldatas)
```

Comment:
This function is used for the tenderly simulation link only. All the changes are based purely on the fact that the args have changed.